modules/sc-mesh-secure-deployment/requirements/server-requirements.txt

```diff
@@ -4,3 +4,4 @@ asymcrypt==0.0.10
 Flask==1.1.2
 getmac==0.8.2
 termcolor
+pandas
```

Comment:
The last time I tried to install it by pip, pandas was failing.
Is it working now?

Reply:
I tried to install pandas in venv and seems to install ok.

(env) mika@station:~/$ pip3 install pandas
Collecting pandas
  Using cached pandas-1.3.1-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.7 MB)
Requirement already satisfied: pytz>=2017.3 in /home/mika/work/fog_drone/github/mesh_com/env/lib/python3.8/site-packages (from pandas) (2021.1)
Requirement already satisfied: python-dateutil>=2.7.3 in /home/mika/work/fog_drone/github/mesh_com/env/lib/python3.8/site-packages (from pandas) (2.8.2)
Requirement already satisfied: numpy>=1.17.3 in /home/mika/work/fog_drone/github/mesh_com/env/lib/python3.8/site-packages (from pandas) (1.21.1)
Requirement already satisfied: six>=1.5 in /home/mika/work/fog_drone/github/mesh_com/env/lib/python3.8/site-packages (from python-dateutil>=2.7.3->pandas) (1.16.0)
Installing collected packages: pandas
Successfully installed pandas-1.3.1